tests/test_inventory.py

```diff
@@ -0,0 +1,48 @@
+"""Tests for the inventory module."""
+
+from io import BytesIO
+from os.path import join
+from pathlib import Path
+
+import pytest
+
+from mkdocstrings.inventory import Inventory, InventoryItem
+
+try:
+    from sphinx.util.inventory import InventoryFile
+except ImportError:
+    InventoryFile = None  # type: ignore
+
+MKDOCSTRINGS_OBJECTS_INV = Path("site/objects.inv")
+
+
+@pytest.fixture(scope="module", autouse=True)
+def _skip_if_sphinx_missing():
+    """Skip test if Sphinx is not installed.
+
+    Auto-applied to all functions in module.
+    """
+    if InventoryFile is None:
+        pytest.skip("Sphinx is not installed")
```

Comment:
I think you can just write `inventory = pytest.importorskip('sphinx.util.inventory')` and you don't need any of the explicit `skip` stuff.

https://docs.pytest.org/en/latest/how-to/skipping.html#skipping-on-a-missing-import-dependency

Reply:
Ah, nice, 1 line instead of 12, I'll try that, thanks.